elide-core/src/main/java/com/yahoo/elide/core/filter/Predicate.java

```diff
@@ -43,6 +44,14 @@ public Predicate(PathElement pathElement, Operator op, List<Object> values) {
         this(Collections.singletonList(pathElement), op, values);
     }
 
+    public Predicate(PathElement pathElement, Operator op){
+        this(Collections.singletonList(pathElement), op, new ArrayList<Object>());
```

Comment:
I believe we do have a constructor for Predicate where it takes a pathElement instead of a list of it